### PR TITLE
PG Search addition to application

### PR DESCRIPTION
**Pull request by trschmitt**
Successfully added pg_search to application. Also added some slight bootstrap styling to index pages.

### CODE REVIEW

File: app/views/questions/index.html.erb [hunk 1]
```diff
@@ -14,8 +14,8 @@
       <tr>
         <td><%= question.title %></td>
         <td><%= question.body %></td>
-        <td><%= link_to 'Show', question %></td>
-        <td><%= link_to 'Delete', question, method: :delete, data: { confirm: "Are you sure?"} %></td>
+        <td><%= link_to 'Show', question, class: "btn btn-primary"%></td>
+        <td><%= link_to 'Delete', question, method: :delete, data: { confirm: "Are you sure?"}, class: "btn btn-warning"%></td>
```
Comment (deansy11) [inline app/views/questions/index.html.erb:18]: Love the button adds

**Review by deansy11 — APPROVED** [review]
Great add of pg_search and results appear to be showing as expected.

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,20 @@
 Rails.application.routes.draw do
+  get 'results/new'
+
+  get 'results/create'
+
+  get 'results/edit'
+
+  get 'results/index'
+
```
Comment (rposborne) [inline config/routes.rb:9]: I don't think we are using these routes.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -11,13 +11,17 @@
...
-              <span><%= link_to "SIGN UP", new_users_path %></span>
-              <span><%= link_to "QUESTIONS", questions_path %></span>
-          </div>
+      <div class="container">
+        <div class="navbar_header">
+            <span><%= link_to "LOGIN", new_session_path%></span>
+            <span><%= link_to "SIGN UP", new_users_path %></span>
+            <span><%= link_to "QUESTIONS", questions_path %></span>
+          <form class="navbar-form" action="/results" method="get">
+            <input type=”text” name=q id=”search” placeholder="Search Questions">
```
Comment (rposborne) [inline app/views/layouts/application.html.erb:20]: Take a look at those quotes.  They are called Smart quotes, and they are the devil. Change them to regular `"` double quotes.

File: test/controllers/results_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class ResultsControllerTest < ActionDispatch::IntegrationTest
+
+  test "should get index" do
+    get results_index_url
```
Comment (rposborne) [inline test/controllers/results_controller_test.rb:6]: Make sure these tests pass.

**Review by rposborne — COMMENTED** [review]
Couple tweaks to improve here but looks good

**Pull request merged**